exporter/device.go

```diff
@@ -0,0 +1,38 @@
+package exporter
+
+import (
+	"context"
+	"time"
+
+	"github.com/ffddorf/unms-exporter/client/devices"
+	"github.com/ffddorf/unms-exporter/models"
+)
+
+type Device struct {
+	*models.DeviceStatusOverview
+}
+
+func (e *Exporter) fetchDeviceData() ([]Device, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	params := &devices.GetDevicesParams{
+		WithInterfaces: &e.wanMetrics,
```

Comment:
this switch is not about WAN, but about statistics for all interfaces. if `WithInterfaces` is false, there won't be metrics for any interface.